### PR TITLE
Feature/jquery plugin

### PR DESCRIPTION
**Pull request by jakiestfu**

### CODE REVIEW

File: dist/jquery.reframe.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// jquery.reframe.js
+// - runs for all selector unless otherwise specified
+// - it JUST creates a fluid wrapper
+(function reframeFunc($) {
+  $.fn.reframe = function (cName) {
+    var classname = typeof cName !== 'undefined' ? cName : 'js-reframe';
```
Comment (jakiestfu) [inline dist/jquery.reframe.js]: `var className = cName || 'js-reframe';

**Comment by yowainwright** [comment]
Hey guys, I think after some awesome tips from @jakiestfu this thing might be fairly close to merging. 

Do you all have any thoughts?

File: bower.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "reframe.js",
   "description": "Reframe.js: responsive iframes for embedded content",
   "main": "index.html",
```
Comment (briangonzalez) [inline bower.json]: Can we make this your js entry point? `dist/reframe.js`

Comment (yowainwright) [inline bower.json]: @briangonzalez done.

**Pull request merged**